README.md

```diff
@@ -2,6 +2,14 @@
 
 This extension provides CiviCRM finance improvement.
 
+This extension supports credit card refund with extension support refund.
+
+The extension must implement PaymentProcessor.get_refundable_balance API to supporting multiple refunds per transaction.
```

Comment:
@kieron-compucorp  please do update the  README.md as it is not true any more, as per our conversation. The payment processor must implement getRefundedAmountByChargeId method in order for this to work.